substra/cli/interface.py

```diff
@@ -1038,6 +1038,27 @@ def run_local(algo, train_opener, test_opener, metrics, rank,
                    inmodels=inmodels)
 
 
+@cli.group()
+@click.pass_context
+def cancel(ctx):
+    """Cancel execution of an asset."""
+    pass
+
+
+@cancel.command('compute_plan')
+@click.argument('compute_plan_id', type=click.STRING)
+@click_global_conf
```

Comment:
it looks the output format option is missing?

Reply:
You're right, I forgot to add it here